### PR TITLE
[Stopwatch] Fix missing deprecations

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | no needed

Fix https://github.com/symfony/symfony/pull/32408

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

File: src/Symfony/Component/HttpKernel/Tests/Debug/TraceableEventDispatcherTest.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ class TraceableEventDispatcherTest extends TestCase
     public function testStopwatchSections()
     {
         $dispatcher = new TraceableEventDispatcher(new EventDispatcher(), $stopwatch = new Stopwatch());
-        $kernel = $this->getHttpKernel($dispatcher, function () { return new Response(); });
+        $kernel = $this->getHttpKernel($dispatcher, function () { return new Response('', 200, ['X-Debug-Token' => '292e1e']); });
```
Comment (jschaedl) [inline src/Symfony/Component/HttpKernel/Tests/Debug/TraceableEventDispatcherTest.php]: Shouldn't this be removed?

**Comment by fabpot** [comment]
Thank you @yceruto.

File: src/Symfony/Component/HttpKernel/Tests/Debug/TraceableEventDispatcherTest.php (hunk 1 above)
Comment (jschaedl) [reply src/Symfony/Component/HttpKernel/Tests/Debug/TraceableEventDispatcherTest.php]: See: https://github.com/symfony/symfony/pull/32431

File: src/Symfony/Component/Stopwatch/Section.php [hunk 2]
```diff
@@ -82,7 +82,7 @@ public function get($id)
      */
     public function open($id)
     {
-        if (null === $session = $this->get($id)) {
+        if (null === $id || null === $session = $this->get($id)) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Stopwatch/Section.php]: I'm reverting this change, it's a BC break caught by the test suite.

Comment (nicolas-grekas) [reply src/Symfony/Component/Stopwatch/Section.php]: hmm ok, see https://github.com/symfony/symfony/pull/32494

**Pull request merged**